ledger-tool/src/output.rs

```diff
@@ -591,25 +700,14 @@ pub fn output_ledger(
     let num_slots = num_slots.unwrap_or(Slot::MAX);
     let mut num_printed = 0;
     let mut all_program_ids = HashMap::new();
-    for (slot, slot_meta) in slot_iterator {
+    for (slot, _slot_meta) in slot_iterator {
         if only_rooted && !blockstore.is_root(slot) {
             continue;
         }
         if slot > ending_slot {
             break;
         }
 
-        match output_format {
-            OutputFormat::Display => {
-                println!("Slot {} root?: {}", slot, blockstore.is_root(slot))
-            }
-            OutputFormat::Json => {
-                serde_json::to_writer(stdout(), &slot_meta)?;
-                stdout().write_all(b",\n")?;
-            }
```

Comment:
In `output_slot()`, we print SlotMeta if we're in `OutputFormat::Display` mode. It felt off to me that we'd print the `SlotMeta` in the other case in here, so removed the piece altogether.

Now, `ledger-tool blockstore print` and `ledger-tool blockstore slot` will have identical output for each slot. Ultimately, I might try to do away with `print` command, but that is for another day / PR